From 5c5daf571a59c88fd75b33105d068813d2f9d92c Mon Sep 17 00:00:00 2001
From: Jiri Kosina <jkosina@suse.cz>
Date: Thu, 2 Sep 2010 16:35:22 +0200
Subject: Revert "drivers/usb: Remove unnecessary return's from void
 functions" partially
Patch-mainline: v2.6.37
Git-commit: 5c5daf571a59c88fd75b33105d068813d2f9d92c

This is a partial revert of 7f26b3a7533b ("drivers/usb: Remove unnecessary
return's from void functions") as this hunk will go through USB tree
due to conflict.

Reported-by: Stephen Rothwell <sfr@canb.auug.org.au>
Signed-off-by: Jiri Kosina <jkosina@suse.cz>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/gadget/langwell_udc.c |    3 +++
 1 file changed, 3 insertions(+)

--- a/drivers/usb/gadget/langwell_udc.c
+++ b/drivers/usb/gadget/langwell_udc.c
@@ -284,6 +284,7 @@ static void ep0_reset(struct langwell_ud
 	}
 
 	VDBG(dev, "<--- %s()\n", __func__);
+	return;
 }
 
 
@@ -1485,6 +1486,7 @@ static void langwell_udc_start(struct la
 	writel(usbcmd, &dev->op_regs->usbcmd);
 
 	DBG(dev, "<--- %s()\n", __func__);
+	return;
 }
 
 
@@ -1507,6 +1509,7 @@ static void langwell_udc_stop(struct lan
 	writel(usbcmd, &dev->op_regs->usbcmd);
 
 	DBG(dev, "<--- %s()\n", __func__);
+	return;
 }
 
 
